scripts.js

```diff
@@ -0,0 +1,20 @@
+var h = prompt ('Podaj wysokośc choinki');
+var star = '';
+
+if (h <= 0 ) {
+	console.log('Choinka nie istnieje');
+	alert('Choinka nie istnieje');
+}
+
+function drawTree(h) {
+	for (var i = 0; i <= h; i++) {
+		star = '';
+		for (var j = 0; j <= i; j++) {
+			star += '*';
+			if (j = i) {
```

Comment:
why this if is here? :) is it necessary?

Reply:
It was in recommendation:
"Wewnętrzna pętla ma się wykonać zależnie od tego, który wiersz aktualnie drukujemy w konsoli. Czyli od tego, która to iteracja głównej pętli. Warunek stopu to aktualny poziom (wartość zmiennej i)."

Reply:
ok, ale `j = i` to nie jest porównanie tylko przypisanie